BundleTransport/app/src/main/java/net/discdd/bundletransport/TransportWifiDirectService.java

```diff
@@ -181,4 +183,8 @@ TransportWifiDirectService getService() {
             return TransportWifiDirectService.this;
         }
     }
+
+    public void setTransportPaths(TransportPaths transportPaths) {
```

Comment:
i don't think we want this. we might end up with a race condition. TransportWifiDirectService should probably make it's own TransportPaths in onStartCommand()